androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/splash/SplashScreen.kt

```diff
@@ -1,78 +1,260 @@
+private val FinalLogoOffset = Offset(0f, -230f)
+private const val InitialLogoScale: Float = 1f
+private const val FinalLogoScale: Float = 0.83f
+private const val InitialBlurRadius: Float = 0f
+private const val FinalBlurRadius = 25f
+private const val InitialAlpha: Float = 0f
+private const val FinalAlpha: Float = 1f
+private const val InitialTopGradientAlpha: Float = 0.2f
+private const val InitialBottomGradientAlpha: Float = 0.6f
+private const val BottomGradientAlphaMultiplier: Float = 1 - InitialBottomGradientAlpha
+
+private const val ForgotTextAlpha = 0.5f
 
 @Composable
 fun SplashScreen() {
-    var shouldShowLogo by remember {
-        mutableStateOf(false)
-    }
+    var shouldShowLogo by remember { mutableStateOf(false) }
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    var logoOffset by remember { mutableStateOf(InitialLogoOffset) }
+    var logoScale by remember { mutableStateOf(InitialLogoScale) }
+    var alpha by remember { mutableStateOf(InitialAlpha) }
+    var blurRadius by remember { mutableStateOf(InitialBlurRadius) }
+    var shouldShowLoginForm by remember { mutableStateOf(false) }
+
+    val floatTweenSpec = tween<Float>(
+        durationMillis = LoginFormRevealDurationInMillis,
+        delayMillis = LogoDurationInMillis
+    )
+    val animateBlur by animateFloatAsState(
+        targetValue = blurRadius,
+        floatTweenSpec
+    )
+    val animateAlpha by animateFloatAsState(
+        targetValue = alpha,
+        floatTweenSpec
+    )
+    val animateLogoScale by animateFloatAsState(
+        targetValue = logoScale,
+        floatTweenSpec
+    )
+    val animateLogoOffset by animateOffsetAsState(
+        targetValue = logoOffset,
+        tween(
+            durationMillis = LoginFormRevealDurationInMillis,
+            delayMillis = LogoDurationInMillis
+        )
+    )
 
     LaunchedEffect(Unit) {
-        delay(DELAY_LOGO)
+        delay(LogoDelayInMillis)
         shouldShowLogo = true
     }
 
-    SplashContent(shouldShowLogo = shouldShowLogo)
+    LaunchedEffect(Unit) {
+        delay(LogoDurationInMillis.toLong())
+        shouldShowLoginForm = true
+        blurRadius = FinalBlurRadius
+        alpha = FinalAlpha
+        logoOffset = FinalLogoOffset
+        logoScale = FinalLogoScale
+    }
+
+    SplashContent(
+        shouldShowLogo = shouldShowLogo,
+        animateAlpha = animateAlpha,
+        animateLogoScale = animateLogoScale,
+        animateLogoOffset = animateLogoOffset,
+        animateBlur = animateBlur
+    )
+
+    if (shouldShowLoginForm) {
+        LoginForm(
+            modifier = Modifier.alpha(animateAlpha),
+            email = email,
+            password = password,
+            onEmailChange = { email = it },
+            onPasswordChange = { password = it },
+            onLogInClick = {
+                // TODO: implement in the integrate PR
+            }
```

Comment:
Then we can reduce the number of parameters here to
```suggestion
        LoginForm(
            modifier = Modifier.alpha(animateAlpha),
            onLogInClick = { email, password ->
                // TODO: implement in the integrate PR
            }
```